exercises/practice/triangle/.docs/instructions.append.md

```diff
@@ -1,10 +1,14 @@
 # About Enum
-The enum statement allows you to create a strongly typed set of labels. You can use that enumeration in the code without having to parse or check for spelling errors.
+The enum statement allows you to create a strongly typed set of labels.
 
```

Comment:
There should only be a new line between new paragraphs

Reply:
Oh I understand it now, I will read the markdown doc later.